docs/gpa.md

```diff
@@ -46,6 +46,15 @@ This document annotates selected parts of the GPA, with details on how to publis
     * Enter the *name of the procuring entity* in its `name`
     * Enter the *address of the procuring entity* in its `address`
     * Enter *other information necessary to contact the procuring entity* in its `contactPoint`
+    * Add a `Classification` object to its `details/classifications` array
+      * Set its `scheme` to 'GPA_COVERAGE_SCHEDULE' ([GPA coverage schedule documentation](https://www.wto.org/english/tratop_e/gproc_e/gp_app_agree_e.htm#revisedGPA))
```

Comment:
For the EU profile, we copied the XML element names, but in general we prefer lower camel case, so:
```suggestion
      * Set its `scheme` to 'gpaCoverageSchedule' ([GPA coverage schedule documentation](https://www.wto.org/english/tratop_e/gproc_e/gp_app_agree_e.htm#revisedGPA))
```